docs/sources/write/shortcodes/index.md

```diff
@@ -1063,6 +1071,85 @@ Insert a simple hero using shortcode arguments:
+
+To add an image map, define the map configuration in the page's front matter and reference it using the shortcode.
+
+### Front matter parameters
+
+Define your image map in the front matter using the `image_maps` parameter.
+
+| Parameter | Description                                                                                    | Required |
+| --------- | ---------------------------------------------------------------------------------------------- | -------- |
+| `key`     | Unique identifier for this image map. Use this value in the shortcode to reference the map.    | yes      |
+| `src`     | Path to the image file. Can be a relative path or full URL.                                    | yes      |
+| `alt`     | Alternative text describing the image for accessibility.                                       | yes      |
+| `points`  | Array of hotspot coordinates and content. Each point defines an interactive area on the image. | yes      |
+
+#### Point parameters
+
+Each item in the `points` array requires the following parameters:
+
+| Parameter | Description                                                                                                 | Required |
+| --------- | ----------------------------------------------------------------------------------------------------------- | -------- |
+| `x_coord` | Horizontal position of the hotspot as a percentage (0-100) of the image width.                              | yes      |
+| `y_coord` | Vertical position of the hotspot as a percentage (0-100) of the image height.                               | yes      |
+| `content` | Markdown content to display in the tooltip. Supports headings, lists, links, and other Markdown formatting. | yes      |
+
+### Shortcode parameters
+
+| Parameter | Description                                                                            | Required |
+| --------- | -------------------------------------------------------------------------------------- | -------- |
+| `key`     | The unique key value from the front matter that identifies which image map to display. | yes      |
+
+### Example
+
+Define the image map in your front matter:
+
+```yaml
+---
+title: My Page
+image_maps:
+  - key: dashboard
+    src: https://grafana.com/static/img/grafana/showcase_visualize.jpg
+    alt: Grafana dashboard
+    points:
+      - x_coord: 22
+        y_coord: 28
+        content: |
+          **Cafe Terrace**
+
+          Small cozy cafe, open 7am–7pm.
+      - x_coord: 64
+        y_coord: 40
+        content: |
+          **City Park**
+
+          Playground, pond, and walking trails.
+      - x_coord: 44
+        y_coord: 72
+        content: |
+          **Historic House**
+
+          Built 1895 — guided tours available.
+---
+```
+
+Reference the image map in your content:
+
+```markdown
+{{</* image-map key="dashboard" */>}}
+```
+
+Produces:
+
+{{< image-map key="dashboard" >}}
```

Comment:
You'll just need to update this image to reflect the update you made

Reply:
Yep once https://github.com/grafana/website/pull/28001 is merged we'll need to publish a new `docs-base` docker image and rebuild the preview.